conf/.env.dev.sample

```
@@ -3,7 +3,7 @@
 # It enables some minimal required functionality
 
 
-BITCART_CRYPTOS=btc,ltc
+BITCART_CRYPTOS=btc,ltc,grs
```

```suggestion
BITCART_CRYPTOS=btc,ltc
```

ltc is used in tests usually